### PR TITLE
Add Random Space Images for Nostr Profile Defaults on Testnet

### PR DESCRIPTION
**Pull request by itailiors**
Set default space-themed images for missing testnet Nostr profile banners and profile pictures.

### CODE REVIEW

File: src/Angor/Client/Pages/Create.razor [hunk 1]
```diff
@@ -724,6 +724,9 @@
 
         try
         {
+            // Ensure default images are set for Testnet
+            SetDefaultImagesForTestnet(project.Metadata);
```
Comment (dangershony) [inline src/Angor/Client/Pages/Create.razor:728]: may I suggest we do that on startup?
also can we also do that for the presets? so a user just goes next next next and invest

File: src/Angor/Client/Pages/Create.razor [hunk 2]
```diff
@@ -1172,4 +1175,45 @@
...
+            if (string.IsNullOrEmpty(metadata.Picture))
+            {
+                metadata.Picture = spacePhotos[random.Next(spacePhotos.Length)];
+            }
+        }
+    }
+    
+    private readonly string[] spacePhotos = new[]
+    {
+        "https://images-assets.nasa.gov/image/hubble-observes-one-of-a-kind-star-nicknamed-nasty_17754652960_o/hubble-observes-one-of-a-kind-star-nicknamed-nasty_17754652960_o~thumb.jpg?as=webp",
```
Comment (DavidGershony) [inline src/Angor/Client/Pages/Create.razor:1199]: Are we allowed to use these images freely?

Comment (itailiors) [reply src/Angor/Client/Pages/Create.razor:1199]: as far as i read from here, it seems like we can.

https://www.nasa.gov/nasa-brand-center/images-and-media/

File: src/Angor/Client/Pages/Create.razor (hunk 1 above)
Comment (itailiors) [reply src/Angor/Client/Pages/Create.razor:728]: When you say startup, you mean when you go into the create-project there will be these links?
And also yeah we can do it for presets, should I make it random or a pre-determined?

Comment (dangershony) [reply src/Angor/Client/Pages/Create.razor]: yeah random put some images on the correct boxes when we land on the page if it is testnet a user can always just change it

File: src/Angor/Client/Pages/Create.razor [hunk 3]
```diff
@@ -616,6 +616,21 @@
                       }
                   };
 
+        // Set default images for Testnet
+        var random = new Random();
```
Comment (dangershony) [inline src/Angor/Client/Pages/Create.razor:620]: whats going on with the spacing here?

File: src/Angor/Client/Pages/Create.razor [hunk 4]
```diff
@@ -625,6 +615,13 @@
         // check if the project was already published to nostr
         nostrMetadataCreated = project.NostrProfileCreated;
         nostrApplicationSpecificDataCreated = !string.IsNullOrEmpty(project.ProjectInfoEventId);
+        
+        // Select a preset
+        if(!project.ProjectInfo.Stages.Any())
```
Comment (dangershony) [inline src/Angor/Client/Pages/Create.razor:620]: calling this so late means the presets dont get saved to local storage but it is ok

**Review by dangershony — APPROVED** [review]

**Pull request merged**